# Panic y Recover

## Esta es la continuacion de panic y recover en funciones
- Para ver el archivo [details](https://github.com/pystudent1913/learning-go/blob/master/04_funciones/details.md)
- Para ver el archivo [panic.go](https://github.com/pystudent1913/learning-go/blob/master/04_funciones/panic.go)

## Contenido
- **panic** muestra la pila de seguimiento de un problema **inesperado**. Por ejemplo ocurrio una **division entre cero**.
- **recover** nos permite continuar la funcionalidad del programa una vez **panic** es invocado.

